tests/optimize/root_finding/test_newton_raphson_hypo.py

```diff
@@ -18,6 +18,7 @@ def less_or_equal(x, compared_to, rel=1e-6):
             or ((x - compared_to) == approx(0.0, rel=rel))
             or (x == approx(x, rel=rel)))
 
+
 nr_func = newton_raphson(1, 1e-6, root=func, return_callable=True)
```

Comment:
...and these are the return_callable use-cases - for times when we only want to jit once, but have the function usable in multiple places (more 'framework' than 'exploratory data science').